mmdeploy/mmcv/ops/nms.py

```diff
@@ -345,7 +345,8 @@ def _multiclass_nms_single(boxes: Tensor,
     labels = labels[:, topk_inds, ...]
 
     if output_index:
-        bbox_index = pre_topk_inds[None, topk_inds]
+        bbox_index = pre_topk_inds[box_inds]
```

Comment:
pre_topk_inds could be None if pre_top_k > 0.

Reply:
Have fixed for pre_top_k<0 or keep_top_k<0.